src/js/techexposure.js

```diff
@@ -3,470 +3,231 @@ import * as d3 from 'd3';
-					: labels.title_opts_how,
-			title_how = typeof labels.port_label === 'undefined' ? 'compared to ' : labels.title_how,
-			caption_market =
-				typeof labels.port_label === 'undefined' ? 'Equity market: ' : labels.caption_market,
-			port_label = typeof labels.port_label === 'undefined' ? 'This portfolio' : labels.port_label,
-			comp_label = typeof labels.comp_label === 'undefined' ? 'Benchmark' : labels.comp_label,
-			hover_over_asset =
-				typeof labels.port_label === 'undefined'
-					? ' of assets under management<br>'
-					: labels.hover_over_asset,
-			hover_over_sec =
-				typeof labels.hover_over_sec === 'undefined'
-					? { before_sec: ' of ', after_sec: ' sector' }
-					: labels.hover_over_sec,
-			hover_over_low_carbon =
-				typeof labels.hover_over_low_carbon === 'undefined'
-					? { before_sec: 'Low-carbon ', after_sec: ' technologies' }
-					: labels.hover_over_low_carbon,
-			lowCarbonLabel =
-				typeof labels.green_legend_label === 'undefined'
-					? { top: 'Low-carbon technologies', bottom: 'within a sector' }
-					: labels.green_legend_label;
-
-		// settings
-		const ttl_width = 700;
-		let margin = { top: 40, right: 20, bottom: 40, left: 20 };
+		this.container = d3.select(container_div);
 
+		// Declare the chart dimensions and margins.
+		const width = 600;
+		const height = 700;
+		const marginTop = 20;
+		const marginRight = 170;
+		const marginBottom = 60;
+		const marginLeft = 90;
 		const bar_width = 30;
 		const bar_gap = 12;
 		const sector_gap = 10;
 		const portfolio_label_offset = 25;
 		const legend_labels_offset = 50;
 
-		// determine left margin based on portfolio name
-		//const portfolio_name = data.filter(d => d.this_portfolio)[0].portfolio_name;
-		const portfolio_name = port_label;
-		let label_width = 0;
-		let test_svg = d3.select(chart_div).append('svg');
-		test_svg
-			.append('text')
-			.text(portfolio_name)
-			.attr('font-size', '10')
-			.each(function () {
-				label_width = this.getBBox().width;
-			});
-		test_svg.remove();
-		margin.left += label_width + portfolio_label_offset;
-
-		label_width = 0;
-		test_svg = d3.select(chart_div).append('svg');
-		test_svg
+		// Labels
+		let port_label = 'This portfolio',
+			comp_label = 'Benchmark*',
+			hover_over_asset = ' of assets under management<br>',
+			hover_over_sec = { before_sec: ' of ', after_sec: ' sector'},
+			hover_over_low_carbon = { before_sec: 'Low-carbon ', after_sec: ' technologies'},
+			lowCarbonLabel = { top: 'Low-carbon technologies', bottom: 'within a sector'};
+
+		// Create the svg container
+		const svg = this.container
+			.append('svg')
+			.attr('width', width)
+			.attr('height', height)
+			.attr('viewBox', [0, 0, width, height])
+			.attr('preserveAspectRatio', 'xMinYMin meet')
+			.attr('style', 'max-width: 100%; height: auto;');
+
+		// Filter the data
+		let equity_market = 'Global Market'; // hardcoded at the moment
```

Comment:
If there is an intention to not hard-code this, but you don't want to deal with it in the PR, then please remove the comment and create a new issue instead. 

In general, I want to avoid us having "do this later" type issues in our code on `main`. These things just end up getting forgotten about/ context lost, and then end up as tech debt in the code.

Reply:
@jdhoffa I adjusted the comment but can also remove it.

Reply:
@jdhoffa the comment is removed

Reply:
Guidance here: 

In the original comment "hard coded at the moment": this indicates that it is hardcoded, but you aren't 100% sure that this is the correct way to do it, and it might need to be considered in the future. That should be a GH issue.

If you are commenting "hard coded", that is also superfluous, as it is clear from the code that the value is hard-coded just by reading the code.